Production_scripts/Process_extracts/Produce_extract_tests/extract_chi_ltc_tests.R

```diff
@@ -0,0 +1,93 @@
+####################################################
+# Name of file - extract_chi_ltc_tests.R
+# Original Authors - Bateman McBride
+# Original Date - August 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               LTC file received from IT.
+#####################################################
+
+year <- "1920"
+
+new_data <- haven::read_sav(get_ltcs_path(year))
+
+# Find and flag any duplicate chis and chi/postcode combinations
+duplicates <- new_data %>%
+  dtplyr::lazy_dt() %>%
```

Comment:
You need to run `usethis::use_package("dtplyr")` to make sure `dtplyr` becomes an import.

Reply:
Done